pkg/scheduler/metrics/metrics.go

```diff
@@ -110,8 +110,11 @@ var (
 	FrameworkExtensionPointDuration *metrics.HistogramVec
 	PluginExecutionDuration         *metrics.HistogramVec
 
-	PermitWaitDuration    *metrics.HistogramVec
+	PermitWaitDuration *metrics.HistogramVec
+	// Deprecated: SchedulerCacheSize is deprecated at v1.33,
```

Comment:
We're deprecating cacheSize.

Reply:
I think the opposite would be easier to understand;
`CacheSize` for "cache_size" (new one) and `SchedulerCacheSize` for "scheduler_cache_side" (old one).

Reply:
opoos... thanks for point out this

Reply:
done